assignments/callbacks.js

```diff
@@ -24,28 +24,56 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(items.length);
```

Comment:
Notice that the getLength function does not specify what array you'll be using. arr is a variable, so the function should be passing in that variable to the callback.
```suggestion
  return cb(arr.length);
```